keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -144,7 +144,6 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
     context("submitting a delivery config manifest") {
       val yamlPayload =
         """---
-        |name: keel-manifest
```

Comment:
Nice easy way to test is to just omit it from the submitted data when calling the controller. I didn't relax the requirement for the `name` to be specified on `DeliveryConfig` itself, only at the edge.